packages/preview-middleware/src/base/config.ts

```diff
@@ -261,16 +262,17 @@ export function createFlpTemplateConfig(config: FlpConfig, manifest: Partial<Man
     const ui5Theme = config.theme ?? (supportedThemes.includes(DEFAULT_THEME) ? DEFAULT_THEME : supportedThemes[0]);
     const id = manifest['sap.app']?.id ?? '';
     const ns = id.replace(/\./g, '/');
+    const basePath = posix.relative(posix.dirname(config.path), '/') ?? '.';
     return {
-        basePath: posix.relative(posix.dirname(config.path), '/') ?? '.',
+        basePath: basePath,
         apps: {},
         init: config.init ? ns + config.init : undefined,
         ui5: {
             libs: getUI5Libs(manifest),
             theme: ui5Theme,
             flex,
             resources: {
-                [PREVIEW_URL.client.ns]: PREVIEW_URL.client.url
+                [PREVIEW_URL.client.ns]: PREVIEW_URL.client.getUrl(basePath)
```

Comment:
An alternative would be to keep PREVIEW_URL as is and add the basePath here
`[PREVIEW_URL.client.ns]: posix.join(basePath, PREVIEW_URL.client.url)`
Not sure which one is better 🤷🏻

Reply:
It's a matter of taste, go for what you prefer.

Reply:
OK then let's leave it as it currently is. This hopefully reduces the risk that someone else just uses the path als URL without basePath